atom/browser/atom_browser_main_parts.cc

```diff
@@ -36,6 +36,7 @@
 #include "chrome/common/chrome_paths.h"
 #include "chrome/common/chrome_result_codes.h"
 #include "chrome/common/chrome_switches.h"
+#include "components/password_manager/core/common/password_manager_features.h"
```

Comment:
is there already a dep for this in the target for atom_browser_main_parts.cc?

Reply:
no, it will fail to find the definition if I remove the header